src/main/java/Main.java

```diff
@@ -0,0 +1,51 @@
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import model.FruitTransaction;
+import service.ParserTransactionsService;
+import service.ProcessDataService;
+import service.ReaderService;
+import service.ReportService;
+import service.WriterService;
+import service.impl.ParserTransactionsServiceImpl;
+import service.impl.ProcessDataServiceImpl;
+import service.impl.ReaderServiceImpl;
+import service.impl.ReportServiceImpl;
+import service.impl.WriterServiceImpl;
+import strategy.handler.BalanceOperationHandlerImpl;
+import strategy.handler.OperationHandler;
+import strategy.handler.PurchaseOperationHandlerImpl;
+import strategy.handler.ReturnOperationHandlerImpl;
+import strategy.handler.SupplyOperationHandlerImpl;
+
+public class Main {
+    public static final String DAILY_OPERATIONS_FILE
+            = "src/main/java/resources/dailyoperations.csv";
+    public static final String DAILY_REPORT_FILE
+            = "src/main/java/resources/dailyreport.csv";
```

Comment:
```suggestion
    public static final String DAILY_OPERATIONS_FILE
            = "src/main/resources/dailyoperations.csv";
    public static final String DAILY_REPORT_FILE
            = "src/main/resources/dailyreport.csv";
```
please be more attentive